apps/user-office-frontend/src/components/user/PeopleTable.tsx

```diff
@@ -277,6 +284,32 @@ const PeopleTable = ({
       onClick: () => setSendUserEmail(true),
     });
 
+  setPrincipalInvestigator &&
+    onRemove &&
+    actionArray.push({
+      icon: () => (
+        <Link
+          href="#"
+          data-cy="assign-as-pi"
+          underline="hover"
+          variant="subtitle2"
+          component="button"
```

Comment:
This line produces an error in the console that the button can not be a child of a button. Maybe try this instead
```suggestion
          onClick={(event) => event.preventDefault()}
```